server/app/database/model_spell_stat_translation.py

```diff
@@ -0,0 +1,23 @@
+import sqlalchemy
+from .base import Base
+from sqlalchemy import Column, String, ForeignKey
+from sqlalchemy.dialects.postgresql import UUID
+
+
+class ModelSpellStatTranslation(Base):
```

Comment:
What is a SpellStat, and why is AoE type in this table? I expect you need translations for spell descriptions, but I don't see a column for that here.

Reply:
Edit: I see the translations for descriptions. What in this table is locale-sensitive?

Reply:
Aoe descriptions need to be localized, right?

Reply:
I feel like it would be best if we are able to associate all of these as the same AoE, e.g. for icons. Do you feel like it's tough?